tests/mock_trello.py

```diff
@@ -1,46 +1,109 @@
-from unittest.mock import patch, MagicMock
-from status import NOT_STARTED, COMPLETED
-from item import Item
-from config import TRELLO_URL, TRELLO_BOARD
-import sys
-import pytest
+from unittest.mock import MagicMock
+from api_client import ApiClient
+from status import NOT_STARTED, COMPLETED, IN_PROGRESS
+from trello import Trello
 
-requestsMock = MagicMock()
+def mock_get(response_body, status_code=200):
+    response = MagicMock()
+    response.json.return_value = response_body
+    response.status_code = status_code
+    return MagicMock(return_value=response)
 
-def mockGetWithResponse(response, status=200):
-    requestsMock.get.reset_mock()
-    responseMock = MagicMock()
-    requestsMock.get.return_value = responseMock
-    responseMock.status_code = status
-    responseMock.json.return_value = response
+def get_mock_trello(trello_config):
```

Comment:
Slightly ambiguous name since you have a class called `Trello`, and this is not a mock `Trello`, it's a real `Trello` with a mocked Trello API (via mocking the api_client).

Also, it might be neater to use e.g. the monkeypatch fixture but what you've done does work.